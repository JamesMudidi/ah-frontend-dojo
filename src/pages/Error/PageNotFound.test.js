import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from 'pages/Error';

// test to check if the Error component matches the snapshot
describe('Login Component', () => {
  it('matches the snapshot', () => {
    const component = shallow(<PageNotFound />);
    expect(component).toMatchSnapshot();
  });
});
