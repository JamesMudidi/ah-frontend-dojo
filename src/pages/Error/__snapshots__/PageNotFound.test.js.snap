// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component matches the snapshot 1`] = `
<div>
  <h1>
    Authors' Haven
  </h1>
  <h4>
    Oops, That page does not exist
  </h4>
</div>
`;
